pkgs/development/python-modules/shap/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, numpy
+, scipy
+, scikitlearn
+, pandas
+, tqdm
+, slicer
+, numba
+}:
+
+buildPythonPackage rec {
+  pname = "shap";
+  version = "0.36.0";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "slundberg";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1wxnxvbz6avzzfqjfbcqd4v879hvpq4021v31fhdpccr2q317rr9";
+  };
+
+  # tests are accessing the network
```

Comment:
Do all of them use network or can we just disable the one or three that fail?

Reply:
@SuperSandro2000 there were tests accessing the network, but also an issue with the C extension not being found during tests, all fixed now, thanks for insisting :)